promptflow/web-classification/run_evaluation.yml

```diff
@@ -9,10 +9,10 @@ column_mapping:
 # define cloud resource
 runtime: automatic
 
-connections:
-  classify_with_llm:
-    connection: Default_AzureOpenAI
-    deployment_name: gpt-35-turbo-0301
-  summarize_text_content:
-    connection: Default_AzureOpenAI
-    deployment_name: gpt-35-turbo-0301
+# connections:
```

Comment:
I commend it out because these are no classify_with_llm and summarize_text_content nodes in classification-accuracy-eval flow